features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql

```diff
@@ -1876,11 +1876,12 @@ CREATE TABLE IF NOT EXISTS SERVICE_CATALOG_ENTRY (
   FOREIGN KEY (CATALOG_ID) REFERENCES SERVICE_CATALOG (ID) ON DELETE CASCADE
 );
 
-CREATE TABLE IF NOT EXISTS AM_API_ENDPOINT_REG_ENTRY_MAPPING (
+create TABLE IF NOT EXISTS AM_API_SERVICE_CATALOG_ENTRY_MAPPING (
```

Comment:
create to uppercase?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/8728/commits/216e08f31573e70a482b6f9e3235b139d06c07bf